packages/mobile/ios/celo.xcodeproj/project.pbxproj

```diff
@@ -847,7 +847,7 @@
 				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = celo/celo.entitlements;
-				CURRENT_PROJECT_VERSION = 34;
+				CURRENT_PROJECT_VERSION = 17;
```

Comment:
Was cutting this in half intentional?

Reply:
No haha this went from 1 -> 2 -> ... 17, but none of those build number changes from me or @cmcewen got merged into master